src/sample.json

```diff
@@ -0,0 +1,82 @@
+[
```

Comment:
Is this file still needed? 
Can it only contain the second array, and be loaded at startup?

import sample from 'sample';